### PR TITLE
Add media type to content provider

### PR DESCRIPTION
**Pull request by dhruvkb**
Since we are now adding new media types, the `ContentProvider` model needs to specify which media type it provides. This PR adds the new `media_type` field to the `ContentProvider` model.

### CODE REVIEW

**Comment by dhruvkb** [comment]
Checking test failure @ `test_stats`

**Review by obulat — COMMENTED** [review]
What about providers with several media types, like Wikimedia or Europeana? Is an array a good option for media_type field?

**Comment by dhruvkb** [comment]
It'll make filtering complicated. Can we not keep two different providers with the same display name but different IDs (like Wikimedia internally stored as wikimedia_images and wikimedia_audio)?

File: openverse-api/catalog/api/constants/media_types.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"""
```
Comment (obulat) [inline openverse-api/catalog/api/constants/media_types.py:1]: Nice to see constants in the API, too!

File: openverse-api/catalog/api/models/models.py [hunk 2]
```diff
@@ -14,6 +15,10 @@ class ContentProvider(models.Model):
         verbose_name='Hide content'
     )
     notes = models.TextField(null=True)
+    media_type = models.CharField(
```
Comment (obulat) [inline openverse-api/catalog/api/models/models.py:18]: I think we should add a node somewhere where we create the content providers about the different internal representation for providers that have several content types (wikimedia_image, wikimedia_audio)

**Review by obulat — APPROVED** [review]
LGTM!

File: openverse-api/catalog/api/models/models.py (hunk 2 above)
Comment (zackkrida) [reply openverse-api/catalog/api/models/models.py:18]: Good suggestion @obulat

**Review by zackkrida — APPROVED** [review]

**Comment by zackkrida** [comment]
> two different providers with the same display name but different IDs

Yeah i think this is fine for now.

File: openverse-api/catalog/api/models/models.py (hunk 2 above)
Comment (dhruvkb) [reply openverse-api/catalog/api/models/models.py:31]: Added usage comment in 99c18e0.

Comment (dhruvkb) [reply openverse-api/catalog/api/models/models.py:31]: Added usage comment in 99c18e0.

**Pull request merged**